test/functional/data.spec.js

```diff
@@ -0,0 +1,52 @@
+const got = require('got');
+const {expect} = require('chai');
+
+let data;
+
+function checkURL(url, head = false) {
+	const method = head ? 'head' : 'get';
+	return got[method](url, {throwHttpErrors: false}).then(({statusCode}) => {
+		if (!head && statusCode === 403) {
+			return checkURL(url, true);
+		}
+
+		expect(statusCode).to.equal(200);
+	});
+}
+
+describe('swag-for-dev', function () {
+	it('data.json is valid', function () {
+		data = require('../../data.json');
+		expect(data).to.be.an('Array');
+		data.forEach(datum => {
+			expect(datum.name).to.be.a('string');
+			expect(datum.difficulty).to.be.oneOf(['easy', 'medium', 'hard']);
+			expect(datum.reference).to.be.a('string');
+			expect(datum.image).to.be.a('string');
+			expect(datum.dateAdded).to.match(/20\d\d-[0-1][\d]-[0-3][\d]T\d\d:\d\d:\d\d\.000Z/);
+			expect(datum.tags).to.be.an('Array').with.length.greaterThan(0);
+		});
+	});
+
+	describe('valid images and references', function () {
+		data = require('../../data.json');
+		delete require.cache[require.resolve('../../data.json')];
```

Comment:
This should be done before calling require. It might be good to add it in the `swag-for-dev.data.json is valid` also

Reply:
I don't think this delete is super necessary now that I think about it. I had initially to make the 2 blocks somewhat idempotent, but if the require fails here, it would fail there as well so it's kind of useless - what do you think?

Reply:
yep in fact it's not necessary from now. let's drop the second require & cachebust

Reply:
Done 👍